ogre_mesh_viewer.py

```diff
@@ -528,11 +540,20 @@ def _toggle_axes(self):
         
         self.axes_visible = not self.axes_visible
 
+    def _toggle_grid(self):
+        self.grid_visible = not self.grid_visible
+
+        if self.grid_visible:
+            self.grid_floor.show_plane(self.fixed_yaw_axis)
+        else:
+            self.grid_floor.show_plane(-1)
+
     def _save_screenshot(self):
         name = os.path.splitext(self.filename)[0]
         outpath = os.path.join(self.filedir, f"screenshot_{name}_")
 
-        Ogre.LogManager.getSingleton().logMessage(f"Screenshot saved to folder: {self.filedir}")
+        Ogre.LogManager.getSingleton().logMessage(f"Screenshot saved to folder: {os.path.normpath(self.filedir)}")
+        print(f"Screenshot saved to folder: {os.path.normpath(self.filedir)}")
```

Comment:
debug leftover?

Reply:
Yes and no, I would like to leave that so that the output is logged to console as well.

But perhaps it should go in another patch